### PR TITLE
Add new gallery example issue templates

### PR DESCRIPTION
**Pull request by ehsteve**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-gallery-example-request.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+name: New gallery example request
+about: This template is for requests for new examples
+title: 'New example to show how to <add words here>'
+labels: new example request
+assignees: ''
```
Comment (Cadair) [inline .github/ISSUE_TEMPLATE/new-gallery-example-request.md]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/new-gallery-example-request.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+name: New gallery example request
+about: This template is for requests for new examples
+title: 'New example to show how to <add words here>'
```
Comment (Cadair) [inline .github/ISSUE_TEMPLATE/new-gallery-example-request.md]:
```suggestion
title: 'Add a new example to show how to <add words here>'
```

File: .github/ISSUE_TEMPLATE/new-gallery-example-request.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+labels: new example request
```
Comment (Cadair) [inline .github/ISSUE_TEMPLATE/new-gallery-example-request.md]: I can't seem to find a reference on what the synatx for this is, I assume this is going to add one label and not three?!

File: .github/ISSUE_TEMPLATE/new-gallery-example-request.md (hunk 1 above)
Comment (Cadair) [reply .github/ISSUE_TEMPLATE/new-gallery-example-request.md]: You don't need to add this if it dosen't do anything, we don't have it in the sunpy ones for instance.

File: .github/ISSUE_TEMPLATE/new-gallery-example-request.md (hunk 3 above)
Comment (ehsteve) [reply .github/ISSUE_TEMPLATE/new-gallery-example-request.md]: I cheated and used the template tool from GitHub so I assume this is right.

Comment (Cadair) [reply .github/ISSUE_TEMPLATE/new-gallery-example-request.md]: cool

**Pull request merged**